wandb/sdk/artifacts/artifact.py

```diff
@@ -103,13 +103,11 @@ class Artifact:
         type: The artifact's type. Use the type of an artifact to both organize
             and differentiate artifacts. You can use any string that contains letters,
             numbers, underscores, hyphens, and dots. Common types include `dataset` or `model`.
-            Include `model` within your type string if you want to link the artifact
-            to the W&B Model Registry.
-        description: A description of the artifact. For Model or Dataset Artifacts,
-            add documentation for your standardized team model or dataset card. View
+            For W&B Registry users, ensure that the artifact type you specify is
+            permitted by the registry you want to link the artifact to.
+        description: A description of the artifact. View
```

Comment:
You can also set the description programmatically - there should also be a setter than we should mention